### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by sushil-mod**
# Added Home Page 

- Navbar component
- Banner component 

## Files to check

- Navbar ( css & jsx )
- Banner ( css & jsx )

##screenshot
note1

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+.gap-sm{
+  gap: 1rem;
+}
+.gap-lg{
+  gap: 4rem;
+}
+.cur-pointer{
+  cursor: pointer;
+}
 /* .App {
```
Comment (sks-21) [inline src/App.css]: Removing commented code is a good practice.

File: src/components/banner/Banner.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    font-weight: 400;
+    margin: 0 22.5rem;
+}
+
+.container__info{
+    width: 30rem;
+    height: 22rem;
+}
+
+.info__title {
```
Comment (sks-21) [inline src/components/banner/Banner.css:22]: Use formatter like Prettier etc to format the code.

File: src/components/banner/Banner.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import './Banner.css';
+
+function Banner() {
+  return <>
+
```
Comment (sks-21) [inline src/components/banner/Banner.jsx]: Same as above

File: src/index.css [hunk 4]
```diff
@@ -1,4 +1,4 @@
-body {
+/* body {
```
Comment (sks-21) [inline src/index.css]: Remove unnecessary commented code.

**Review by sks-21 — APPROVED** [review]
Just check the comments once. Rest everything looks good.

File: src/App.css [hunk 5]
```diff
@@ -1,3 +1,31 @@
...
+.pri-color{
+  color:var(--pri-color);
+}
+.pri-bg-color{
+  background-color: var(--pri-color);
+}
+.text-center{
+  text-align: center;
+}
+.gap-sm{
```
Comment (ultronjs) [inline src/App.css]: You must be having these utility classes in the component library.Can import the same and use it directly

**Review by ultronjs — APPROVED** [review]
The file Structure looks clean
Same thing remove commented Code.
Good to merge
Just a suggestion as Landing home is on fullscreen use box-sizing: border-box that will remove the necessary scrollbar that is coming in the preview as there is not need of scrollbar in the landing Page

**Review by Nagashree01 — APPROVED** [review]
Folder structure is Good. Code is clean and readable. Just remove the commented css code in index.css if you are not using it further. Good to merge

File: src/App.css (hunk 5 above)
Comment (sushil-mod) [reply src/App.css]: yeah , but i forgot these things in CL ( maybe in future refactore and do it ) thanks for suggesting

File: src/index.css (hunk 4 above)
Comment (sushil-mod) [reply src/index.css]: done

File: src/components/banner/Banner.jsx (hunk 3 above)
Comment (sushil-mod) [reply src/components/banner/Banner.jsx]: done

File: src/components/banner/Banner.css (hunk 2 above)
Comment (sushil-mod) [reply src/components/banner/Banner.css:22]: ok i will use

File: src/App.css (hunk 1 above)
Comment (sushil-mod) [reply src/App.css]: done

**Pull request merged**